3.Linear_Modeling/1_2.fit_linear_model_2b.ipynb

```diff
@@ -0,0 +1,294 @@
+{
```

Comment:
<p>I like it how you described the components of the model.</p><p> </p><p>This might be a typo:</p><blockquote>Inhibitor, and Inhibitor Dose. </blockquote><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>